src/commands/waitaof.json

```
@@ -0,0 +1,34 @@
+{
+    "WAITAOF": {
+        "summary": "Wait for of all the write commands sent in the context of the current connection on either local host or replicas to get synced to AOF file on the disk",
```

I prefer short summaries. We can expand in the man page.
```suggestion
        "summary": "Wait until all write commands are synced to AOF",
```

---

your text doesn't mention the local connection (so it's not technically accurate).
also, why drop the part about replicas? (plain view of that text makes it look like it's just the local aof file)

---

My $.02 on a compromise:

> Wait for all write commands sent in the context of the current connection to be synced to AOF of local host and/or replicas